src/node_url.cc

```diff
@@ -1653,9 +1653,10 @@ void URL::Parse(const char* input,
               state = kFragment;
               break;
             default:
-              if ((!IsWindowsDriveLetter(ch, p[1]) ||
-                   end - p == 1 ||
-                   (p[2] != '/' &&
+              if ((end - p == 1 ||
```

Comment:
Depending on the spec writer's intention (https://github.com/whatwg/url/issues/308) changing `end - p == 1` to `end - p <= 1` might be more robust to cover `ch == kEOL` as well, but in any case it shouldn't matter since the case when `ch == kEOL` is covered in another `case` already. Also, nice catch on noticing the `end - p == 1` in the original code should actually be `end - p == 2` :+1: